### PR TITLE
Martin/feature/sp 7

### LINKED ISSUE #7: save astronauts to repository

### PR DESCRIPTION
**Pull request by MartinBlazek6**

### CODE REVIEW

File: src/main/java/com/example/SpaceProject/controller/Controller.java [hunk 1]
```diff
@@ -1,17 +1,64 @@
...
+
+            String[] nameParts = name.split(" ");
+            String firstName = nameParts[0];
+            String lastName = nameParts[1];
+
+            Astronaut astronaut = Astronaut.builder()
+                    .firstName(firstName)
+                    .lastName(lastName)
+                    .build();
+            astronautList.add(astronaut);
```
Comment (MartinBlazek6) [inline src/main/java/com/example/SpaceProject/controller/Controller.java:71]: toto by som dal do servisu

File: src/main/java/com/example/SpaceProject/entity/Astronaut.java [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 @Entity
 @Data
 @NoArgsConstructor
+@AllArgsConstructor
+@Builder
 public class Astronaut {
 
     @Id
-    @Generated
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (MartinBlazek6) [inline src/main/java/com/example/SpaceProject/entity/Astronaut.java:18]: preco je to takto

Comment (MartinBlazek6) [reply src/main/java/com/example/SpaceProject/entity/Astronaut.java:18]: lebo

File: src/main/java/com/example/SpaceProject/controller/Controller.java (hunk 1 above)
Comment (MartinBlazek6) [reply src/main/java/com/example/SpaceProject/controller/Controller.java:71]: dobre tak to dam do servisu

**Review by Kakika85 — APPROVED** [review]

**Review by Podkulova — APPROVED** [review]

**Pull request merged**